src/qbittorrent.exe.manifest

```diff
@@ -37,9 +37,10 @@
   </compatibility>
 
   <!-- Enable long paths that exceed MAX_PATH in length -->
```

Comment:
Last comment. Please move this comment to just above `<longPathAware>`. And maybe add a comment for `<heapType>` too.

Reply:
@Chocobo1 Should we also follow Microsoft's guide for consistency in relation to `<longPathAware>` ??

`<ws2:longPathAware xmlns:ws2="http://schemas.microsoft.com/SMI/2016/WindowsSettings">true</ws2:longPathAware>`

https://learn.microsoft.com/en-au/windows/win32/sbscs/application-manifests#longpathaware

Reply:
>@Chocobo1 Should we also follow Microsoft's guide for consistency in relation to <longPathAware> ??

To be honest, I don't have strong opinion about it.